salt/modules/swarm.py

```diff
@@ -29,6 +29,10 @@
 try:
     import docker
     HAS_DOCKER = True
+    def __init__(self):
```

Comment:
Probably better to just check `HAS_DOCKER` in the init function.

Reply:
@gtmanfred Done